### PR TITLE
Issue #2470918: Create hook_help

### PR DESCRIPTION
**Pull request by morvans**
This PR is basically the patch at https://www.drupal.org/node/2470918#comment-10236761
It replaces PR167 https://github.com/fago/rules/pull/167

### CODE REVIEW

File: rules.module [hunk 1]
```diff
@@ -98,3 +99,45 @@ function rules_entity_update(EntityInterface $entity) {
...
+/**
+ * Implements hook_help().
+ */
+function rules_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    case 'help.page.rules':
+      $output = '';
+      $output .= '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('Rules documentation is kept online. Please use the links below for more information about Rules. Feel free to contribute to improving the online documentation! For more information, see the <a href="!action" target="_blank">online documentation for the Rules module.</a>',
+              array('!action' => 'https://www.drupal.org/project/rules')) . '</p>';
```
Comment (klausi) [inline rules.module]: please use the short array syntax. And the "!" placeholder should not be used anymore, see https://www.drupal.org/node/2506427

File: rules.module [hunk 2]
```diff
@@ -18,11 +18,21 @@ use Drupal\rules\Event\UserLogoutEvent;
  */
 function rules_help($route_name, RouteMatchInterface $route_match) {
   switch ($route_name) {
-    case 'entity.rules_reaction_rule.collection':
-      $output = t('Reaction rules, listed below, react on selected events on the site. Each reaction rule may fire any number of <em>actions</em>, and may have any number of <em>conditions</em> that must be met for the actions to be executed. You can also set up <a href="!url1">components</a> – stand-alone sets of Rules configuration that can be used in Rules and other parts of your site. See <a href="!url2">the online documentation</a> for an introduction on how to use Rules.', [
```
Comment (klausi) [inline rules.module:22]: Hm, this should stay, right?

Comment (morvans) [inline rules.module:22]: Erm... Yes :/ I'll take a look at that one

**Comment by morvans** [comment]
I'm not sure if the failure is really related to my code...

**Comment by klausi** [comment]
replaced by #325

**Pull request closed without merging**